pkg/config/config_unix.go

```diff
@@ -38,7 +38,8 @@ func DefaultConfig() PluginConfig {
 			NoPivot:            false,
 			Runtimes: map[string]Runtime{
 				"runc": {
-					Type: "io.containerd.runc.v1",
+					Type:                 "io.containerd.runc.v1",
+					ContainerAnnotations: []string{"io.kubernetes.*", "io.k8s.*"},
```

Comment:
Let's not put any default here.

If we really want to have a default value, I hope it is the device plugin domain we want to push, e.g. `io.kubernetes.device-plugin.*`.

Reply:
> Let's not put any default here.

done